docs/Interoperability_tests_guide/Running_test_cases.md

```diff
@@ -0,0 +1,82 @@
+# Running test cases
+
+If you have test cases configured in the platform, you can run them on your device using the **Protocol tests** panel. Read this chapter to learn how to display test case descriptions, start and stop test case execution and view test results and logs.
+
+## Protocol tests panel description
+
+The protocol tests panel is available in the **Device Management Center** individually for each device. To access it, select a device in **Device Inventory** to enter its **Device Management Center** and select **Protocol tests** from the menu on the left.   
```

Comment:
Very useful tip - use the settings button under the menu to add protocol tests option - sometimes (depends on configuration) user might not see the protocol tests option in menu.